### ISSUE TITLE
Library not mocked by jest's "moduleNameMapper" due to lack of .css in module name

### ISSUE THREAD

**Issue by gi-alec** [KyleAMathews/typefaces#113]
Due to the lack of `.css` in the module name, the import is not mocked and fails the tests. This is not a problem in `normalize.css` for example.

A typical jest configuration will use library `identity-obj-proxy` with a glob like `.+\\.(css|styl|less|sass|scss)` to mock css imports from node_modules.

I was able to resolve the problem by specifying the location of the css file.
`import 'typeface-montserrat/index.css'`

Seems there should be some documentation for this special case for jest users due to the popularity of the library. Do advise if you feel there is a better way to handle the problem.

Cheers

**Comment by gibbok** [comment]
Yes, it should be definitely documented under "jest integration". Thanks for sharing, it works.

Here my jest.config.js

```json
  moduleNameMapper: {
    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
      '<rootDir>/__mocks__/fileMock.js',
    '\\.(css|less|sass|scss)$': 'identity-obj-proxy',
  }
```
